ONSdigital/dp-kafka
# Feature/tls not env vars

### What

lib no longer uses env vars, but is passed a nil or otherwise security config

### How to review

- connect to non-TLS and TLS (examples)
- see the topic-manager (https://github.com/ONSdigital/dp-setup/pull/1204) for an app using the admin features
- allow env vars for key/cert to be either file paths or (now) contain the key/cert themselves

### Who can review

!gedge
